contracts/VerifierRegistry.sol

```diff
@@ -151,4 +169,13 @@ contract VerifierRegistry is Ownable {
     emit LogUpdateActiveStatus(msg.sender, _verifierAddress, _active);
   }
 
+  function hashName(string _base) internal pure returns (bytes32) {
```

Comment:
maybe it will be good idea to also `trim` the `_base`?

Reply:
depends on what you want to do. we only allow unique/caseinsensitive  names

Reply:
name will be displaying via web and HTML ignores spaces, so we can at least trim the string to make it more user friendly if we already using case insensitive, if not, we will need to remember to display all spaces in UI... well its not big deal... just an idea.